tcpdump.c

```diff
@@ -1162,17 +1162,27 @@ _U_
 		free(host_url);
 	} else
 #endif
-	status = pcap_findalldevs(&devlist, ebuf);
-	if (status < 0)
+    status = pcap_findalldevs(&devlist, ebuf);
```

Comment:
Please don't change indentation.

Reply:
I apologize, my linter took over for me on that one. I hope this is better!